sdk/src/main/java/com/google/cloud/dataflow/sdk/runners/inprocess/InProcessPipelineRunner.java

```diff
@@ -189,6 +208,14 @@ public InProcessPipelineOptions getPipelineOptions() {
     return options;
   }
 
+  Supplier<Clock> getClockSupplier() {
+    return clockSupplier;
+  }
+
+  void setClockSupplier(Supplier<Clock> supplier) {
```

Comment:
I worry about changes in behavior when the clock is changed midway through processing. Why does it need to be mutable?